internal/daemon/daemon.go

```diff
@@ -780,6 +780,19 @@ func (d *Daemon) RebootIsMissing(st *state.State) error {
 	return errExpectedReboot
 }
 
+// PassServiceArgs passes the provided service arguments to
+// the service manager responsible for services under daemon overlord.
+func (d *Daemon) PassServiceArgs(serviceArgs map[string][]string) error {
+	if d.overlord == nil {
+		return fmt.Errorf("internal error: no Overlord")
+	}
+	serviceMgr := d.overlord.ServiceManager()
+	if serviceMgr == nil {
+		return fmt.Errorf("internal error: no Service Manager")
```

Comment:
Same here.  The ServiceManager is "contractually" defined to exist there. If it doesn't, everything is _really_ broken and the panic will help us more than the error message.